tchannel-core/pom.xml

```diff
@@ -62,6 +62,11 @@
             <artifactId>libthrift</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <scope>provided</scope>
```

Comment:
Cab provided be inherited?

Reply:
specifying it here is more obvious/fail-safe.